website/docs/cloud-docs/users-teams-organizations/organizations.mdx

```diff
@@ -23,7 +23,7 @@ In addition to the Terraform Cloud UI, you can use the following methods to mana
 
 ## Selecting Organizations
 
-Terraform Cloud displays your current organization in the left of the top navigation bar. To select an organization:
+Terraform Cloud displays your current organization in the bottom left. To select an organization:
```

Comment:
Also, leave this for Judith to see --> Judith, we went back and forth on this. On the one hand, we don't like having any positional language. But on the other hand, I had trouble finding where the organization was listed when I first saw the UI, as it's far at the bottom where I typically don't look and not grouped with anything else in the navigation bar. So I was thinking in this one instance where we talk about how to select, it could be nice to help people find it. But maybe this isn't necessary. Let me know your opinion and we can take it out if you think it's not needed.
```suggestion
Terraform Cloud displays your current organization in the bottom left of the sidebar. To select an organization:
```